data/json/items/tool/lighting.json

```diff
@@ -1081,5 +1081,52 @@
       { "type": "firestarter", "moves": 100 }
     ],
     "flags": [ "LIGHT_15", "TRADER_AVOID", "ALLOWS_REMOTE_USE", "FIRESTARTER", "FIRE" ]
+  },
+  {
+    "id": "hobo_candle",
+    "type": "TOOL",
+    "name": { "str": "hobo candle" },
+    "looks_like": "jar_3l_glass_sealed",
+    "description": "An gasoline based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  You'll need a lighter or matches to light it.",
```

Comment:
```suggestion
    "description": "A gasoline-based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  You'll need a lighter or matches to light it.",
```